dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/search/TestSolrClientFactory.java

```diff
@@ -24,13 +25,14 @@ public class TestSolrClientFactory extends SolrClientFactory {
     private static final Logger LOGGER = Logger.getLogger(TestSolrClientFactory.class.getCanonicalName());
 
     private static final String DEFAULT_SOLR_TEST_PORT = "8984";
+    private static final String DEFAULT_SOLR_TEST_HOST = "localhost";
 
     // -------------------- LOGIC --------------------
     
     @Produces
     @Specializes
     public SolrClient produceSolrClient() throws IOException {
-        String urlString = "http://localhost:" + resolveSolrPort() + "/solr/collection1";
+        String urlString = resolveSolrURL() + "/solr/collection1";
```

Comment:
I'm not a fan of magic strings in code. Maybe we could extract it to constant? Same for one below

Reply:
Fair point, extracted to constants.